linux_os/guide/system/permissions/mounting/kernel_module_usb-storage_disabled/rule.yml

```
@@ -40,6 +40,8 @@ references:
     cobit5: APO13.01,DSS01.04,DSS05.03,DSS05.04,DSS05.05,DSS05.07,DSS05.10,DSS06.03,DSS06.10
     iso27001-2013: A.11.2.6,A.13.1.1,A.13.2.1,A.18.1.4,A.6.2.1,A.6.2.2,A.7.1.1,A.9.2.1,A.9.2.2,A.9.2.3,A.9.2.4,A.9.2.6,A.9.3.1,A.9.4.2,A.9.4.3
     cis-csc: 1,12,15,16,5
+    cis: 8.4,8.5
```

```suggestion
    cis@rhel7: 8.4,8.5
```
The build system doesn't allow `cis` and `cis@rhel8` at the same time. Are these identifiers specific to `rhel7` instead?

---

CIS has base mappings, which is what those were... and then they have mappings that are specific to a baseline.

Made up example could be:
CIS 8.1: Use encryption

RHEL 8 CIS 123.1: Use the following ciphers in sshd config

---

remapped to rhel8